### ISSUE TITLE
r_a_c installer failing on PG gem

### ISSUE THREAD

**Issue by marcamillion** [RailsApps/rails_apps_composer#181]
I have PostgreSQL installed locally - via http://postgresapp.com/ (and it has worked for other projects).

But when I am running through the dialogues, I get this error:

```
Installing pg (0.15.1) 
Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.
compiling pg.c
pg.c: In function ‘pg_s_library_version’:
pg.c:273: warning: implicit declaration of function ‘PQlibVersion’
pg.c: In function ‘Init_pg_ext’:
pg.c:384: error: ‘PQPING_OK’ undeclared (first use in this function)
pg.c:384: error: (Each undeclared identifier is reported only once
pg.c:384: error: for each function it appears in.)
pg.c:386: error: ‘PQPING_REJECT’ undeclared (first use in this function)
pg.c:388: error: ‘PQPING_NO_RESPONSE’ undeclared (first use in this function)
pg.c:390: error: ‘PQPING_NO_ATTEMPT’ undeclared (first use in this function)
make: *** [pg.o] Error 1


Gem files will remain installed in/.rvm/gems/ruby-1.9.3-p194@hcv/gems/pg-0.15.1 for inspection.
Results logged to /.rvm/gems/ruby-1.9.3-p194@hcv/gems/pg-0.15.1/ext/gem_make.out

An error occurred while installing pg (0.15.1), and Bundler cannot continue.
Make sure that `gem install pg -v '0.15.1'` succeeds before bundling.
    composer  Running 'after bundler' callbacks.
Could not find gem 'rails (= 3.2.13) ruby' in the gems available on this machine.
Run `bundle install` to install missing gems.
```

Note that I am using `Rails 3.2.13` in this version....and not Rails 4 as indicated by another issue I just filed.
